test/unit/lib/plugins/aws/package/compile/events/schedule.test.js

```diff
@@ -223,9 +272,130 @@ describe('test/unit/lib/plugins/aws/package/compile/events/schedule.test.js', ()
       InputTemplate: '{"time": <eventTime>, "key": "value"}',
       InputPathsMap: { eventTime: '$.time' },
     });
+    expect(scheduleCfResources[8].Properties.Target.InputTransformer).to.be.undefined;
+    expect(scheduleCfResources[9].Properties.Target.InputTransformer).to.be.undefined;
+  });
+
+  it('should throw when passing "inputPath" or "inputTransformer" to method:schedule resources', async () => {
```

Comment:
This `it` should not be in context of this `describe` (e.g. it doesn't require the `before` job as configured here).

I think to have it clean, we need to reorganize this describe:
1. Move tests that confirm on the outcome of the first `run`, into nested `describe('basic', () => ... )`  block
2. Reorganize following tests as follows:
   1. If outcome of `run` is tested in context of one `it` block, it can remain to be configured with one `it`
   2. If outcome of `run` is tested with multiple `it` blocks, let's wrap it into `describe`

Reply:
I will reorganize these tests.